cypress/component/LibraryCard/LibraryCardTable.spec.tsx

```diff
@@ -14,69 +13,60 @@ describe('Library Card Table Tests', () => {
       userEmail: 'test.user.1@test.com',
       createUserId: 2,
       createDate: new Date(),
+    },
+    {
+      id: 2,
+      userId: 2,
+      userName: 'Test User 2',
+      userEmail: 'test.user.2@test.com',
+      createUserId: 2,
+      createDate: new Date(),
+    },
+    {
+      id: 3,
+      userId: 3,
+      userName: 'Test User 3',
+      userEmail: 'test.user.3@test.com',
+      createUserId: 2,
+      createDate: new Date(),
     }
   ]
 
-  const userOptions = [
-    {userId: 1, displayName: 'Test User 1', email: 'user1@test.com', libraryCard: undefined},
-    {userId: 2, displayName: 'Test User 2', email: 'user2@test.com', libraryCard: undefined},
-    {userId: 3, displayName: 'Test User 3', email: 'user3@test.com', libraryCard: undefined}
-  ]
-
   beforeEach(() => {
     cy.viewport(1000, 800);
   });
 
-  it('Should render the Library Card Table', () => {
+  it('should render the Library Card Table with a list of users', () => {
     const props: LibraryCardTableProps = {
-      libraryCards: libraryCardList,
-      users: [],
+      libraryCards: libraryCardList
     }
     mount(<LibraryCardTable {...props}/>);
     cy.get('[data-cy=manage-library-card-table]').should('exist');
-    cy.get('[data-cy=add-library-card-button]').should('exist');
-    // For each user in the list, teest that the row is deletable and removed from the table view
+    // For each user in the list, test that the row is visible
```

Comment:
I just split this test into two smaller/more focused tests: one to test rendering the library card list, the other to test deleting a library card from the list